deployments/oathkeeper/rules/rules.yaml

```diff
@@ -0,0 +1,39 @@
+- id: workday-everything
+  upstream:
+    url: https://host.docker.internal:8080
+  version: v0.40.3
+  match:
+    url: https://workday.flock.community/<.*>
+    methods:
+      - GET
+      - POST
+      - PUT
+      - DELETE
+  authenticators:
+    - handler: noop
+  authorizer:
+    handler: allow
+  mutators:
+    - handler: noop
+  errors:
+    - handler: json
+
+- id: shadow-workday-everything
+  upstream:
+    url: https://host.docker.internal:8080
+  version: v0.40.3
+  match:
+    url: https://shadow.workday.flock.community/<.*>
```

Comment:
We need the upstream url of the shadow version of workday, which is the default url it gets assigned when the google cloud run was created. Something like: `https://shadow-workday-app-tveu425laa-ew.a.run.app`.

Reply:
Updated with `https://workday-app-ory-x6nsuevzyq-ew.a.run.app/` and `ory.workday.flock.community`